app/views/helpers/FullResultsPageHelper.scala

```diff
@@ -389,9 +392,18 @@ object FullResultsPageHelper extends ViewHelper {
           rows = Seq(
             Seq(
               TableRow(content = HtmlContent(p(messages("fullResultsPage.taxableProfit.daysAllocated")))),
-              TableRow(content = HtmlContent(d1.days.toString), classes = "govuk-table__cell--numeric"),
-              TableRow(content = HtmlContent(d2.days.toString), classes = "govuk-table__cell--numeric"),
-              TableRow(content = HtmlContent(totalDays.toString), classes = "govuk-table__cell--numeric")
+              TableRow(
+                content = HtmlContent(s"""${d1.days.toString} <span class="sr-only">${messages("resultsPage.days")}</span>"""),
```

Comment:
The span code for screen reader is duplicated a lot. Can we not add an implicit class to simplify this?